akka-cluster-typed/src/test/scala/akka/cluster/typed/ActorRefIgnoreSerializationSpec.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.cluster.typed
+
+import akka.actor.IgnoreActorRef
+import akka.actor.typed.{ ActorRef, ActorRefResolver, ActorSystem }
+import akka.actor.typed.scaladsl.Behaviors
+import com.typesafe.config.ConfigFactory
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class ActorRefIgnoreSerializationSpec extends AnyWordSpec with ScalaFutures with Matchers {
+
+  "ActorRef.ignore" should {
+
+    "return a serializable ActorRef that can be sent between two ActorSystems using remote" in {
+
+      val config = ConfigFactory.parseString(s"""
+        akka {
+          loglevel = debug
+          actor.provider = cluster
+          remote.classic.netty.tcp.port = 0
+          remote.artery {
+            canonical {
+              hostname = 127.0.0.1
+              port = 0
+            }
+          }
+        }
+        """)
+
+      val system1 = ActorSystem(Behaviors.empty[String], "sys1", config)
+      val system2 = ActorSystem(Behaviors.empty[String], "sys2", config)
+
+      try {
+
+        val remoteRefStr = ActorRefResolver(system1).toSerializationFormat(ActorRef.ignore[String])
+        // the remote ref string path must be the actor sys name with host and port
+        // concatenated with IgnoreActorRef relative path
+        remoteRefStr shouldBe IgnoreActorRef.actorPath(system1.address.hostPort).toString
+
+        val ref: ActorRef[String] = ActorRefResolver(system2).resolveActorRef[String](remoteRefStr)
+        ref.path.toString shouldBe IgnoreActorRef.actorPath(system2.address.hostPort).toString
```

Comment:
When it is a single or actor-system single instance, also compare instance.

Reply:
I don't get your point here.

Reply:
If you deserialize ignore from another system it should be the same instance of ignore as in this system, not create a new object every time - feedback above, so should be verified here when that is the case.

Reply:
This is not possible because of the trick I did [here](https://github.com/akka/akka/pull/28630/files/9490cc8e6eb8d42b983ba91c429ef5878ef43eda#r382329107) to avoid sending messages over the wire, but also because `ActorRef.ignore` doesn't depend on an ActorSystem.

However, we can prove that: 
 * `ActorRef.ignore[String] isSameAs ActorRef.ignore[String]`
 * `deserRef isSameAs otherDeserRef` becasue the deserialized instance is cached.